content/zh-cn/docs/tasks/run-application/run-replicated-stateful-application.md

```diff
@@ -124,11 +126,11 @@ ConfigMap 本身没有什么特别之处，因而也不会出现不同部分应
 每个 Pod 都会在初始化时基于 StatefulSet 控制器提供的信息决定要查看的部分。
 
 <!--
-### Services 
+### Create Services {#services}
 
 Create the Services from the following YAML configuration file: 
 -->
-### 服务  {#services}
+### 创建服务  {#services}
```

Comment:
Would it help to write “Service” here (the name of the API kind that the reader must create)? The capital S in Services is a hint there that the page is referring to the API not the English word.